extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -787,6 +787,12 @@ export class AmpAutocomplete extends AMP.BaseElement {
           this.displayUserInput_();
           this.toggleResults_(false);
         });
+      case Keys.TAB:
+        if (this.activeElement_) {
+          this.fireSelectEvent_(this.userInput_ =
```

Comment:
nit: let's split this into two lines. Assignment in function call arguments can be a code smell